cwms-data-api/src/test/resources/cwms/cda/data/dto/timeseriesprofile/timeseriesprofileinstance.json

```diff
@@ -0,0 +1,44 @@
+{
+  "time-series-profile":
+  {
+    "location-id": "DET",
+    "key-parameter": "Depth",
+    "parameter-list": [ "Temp-Water", "Depth"],
+    "description": "Demonstration Profile for Couger Lake",
+    "office-id": "SWT",
+    "ref-ts-id": "DET.Elev-Forebay.Inst.0.0.Mixed-Rev"
+  },
+
+  "timeseries-list" :  [
+    {
+         "parameter": "Temp-Water",
+         "unit": "F",
+         "time-value-pairs":
+        [
+           {
+              "time": "09/09/2019,13:16:01Z",
```

Comment:
As a TODO: this should be in the ISO8601 format which is YYYY-MM-DDTHH:MM:SS.sss[Timezone in some format].

This is fine for demonstration purposes though.